samples/extensions/dynamic_blending/README.adoc

```diff
@@ -0,0 +1,67 @@
+////
+- Copyright (c) 2023, Mobica Limited
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+////
+
+== Dynamic blending
+
+=== Overview
+
+This sample demonstrates the functionality of VK_EXT_extended_dynamic_state3 related to blending. It includes the
+following features: 
+
+* `+vkCmdSetColorBlendEnableEXT+`: toggles blendingon and off.
```

Comment:
blendingon -> blending on
... and what do all those `+` around the coded words mean? Are they some adoc format specific stuff?

Reply:
Exactly, that's asciidoc syntax. Though I don't think you don't need to have both the ` and the + 

One should suffice.

Reply:
I used a automatic converter to turn the markdown file that I already had into the asciidoc. It transformed it in this way, and I assumed it should be like that, because it looked good in the preview. I corrected that.